zerver/lib/event_queue.py

```diff
@@ -125,6 +133,7 @@ def add_event(self, event):
         self.finish_current_handler()
 
     def finish_current_handler(self, need_timeout=False):
+        # type: (Optional[bool]) -> bool
```

Comment:
ah, yeah i had assumed `Optional[T]` meant it was optional to the caller of the function, since the kwarg would set it to a default if not provided.